src/index.html

```diff
@@ -12,7 +12,68 @@
       href="./style.css"
     />
   </head>
+  
   <body>
-    <h1>Stars</h1>
+    
+    <div class="stars stars--0">
+      
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      
+    </div>
```

Comment:
Add an empty line between sibling blocks of HTML for readability, but do not add them between parent and child elements.